fe/fe-core/src/main/java/com/starrocks/sql/analyzer/AnalyzeStmtAnalyzer.java

```diff
@@ -138,5 +142,22 @@ public Void visitCreateAnalyzeJobStatement(CreateAnalyzeJobStmt statement, Conne
             }
             return null;
         }
+
+        void analyzeAnalyzeTypeDesc(ConnectContext session, AnalyzeStmt statement, AnalyzeTypeDesc analyzeTypeDesc) {
+            if (analyzeTypeDesc instanceof AnalyzeHistogramDesc) {
+                List<String> columns = statement.getColumnNames();
+                OlapTable analyzeTable = (OlapTable) MetaUtils.getStarRocksTable(session, statement.getTableName());
+
+                for (String columnName : columns) {
+                    Column column = analyzeTable.getColumn(columnName);
+                    if (column.getType().isComplexType()
```

Comment:
How about move this if statement to `Type.java`, and naming it as `Type::supportHistogram`?

Reply:
I think these types of judgments are abstract enough, and they can be used for combination. It is not necessary to put all type combinations into Type.java